include/beast/websocket/teardown.hpp

```diff
@@ -56,12 +77,63 @@ teardown(Socket& socket, error_code& ec) = delete;
     manner equivalent to using boost::asio::io_service::post().
 
 */
+/*
+    If you are trying to use OpenSSL and get an error about this function
+    being deleted, you need to include <beast/websocket/ssl.hpp> in the
+    corresponding source file!
+*/
```

Comment:
This comment can be removed. The function is not deleted anymore, and you have a similar comment for the `static_assert`.